coderd/database/dbauthz/dbauthz_test.go

```diff
@@ -987,18 +987,12 @@ func (s *MethodTestSuite) TestOrganization() {
 	}))
 	s.Run("PaginatedOrganizationMembers", s.Subtest(func(db database.Store, check *expects) {
 		o := dbgen.Organization(s.T(), db, database.Organization{})
-		u := dbgen.User(s.T(), db, database.User{})
-		mem := dbgen.OrganizationMember(s.T(), db, database.OrganizationMember{
-			OrganizationID: o.ID,
-			UserID:         u.ID,
-			Roles:          []string{rbac.RoleOrgAdmin()},
-		})
 
 		check.Args(database.PaginatedOrganizationMembersParams{
-			OrganizationID: uuid.UUID{},
+			OrganizationID: o.ID,
 			LimitOpt:       1,
```

Comment:
I think this different check means we also don't need this field any more

Reply:
Accidentally pushed some test code in the test so not sure which one you were looking at but that's needed.